src/configuration.ts

```diff
@@ -1635,10 +1628,7 @@ export async function setLaunchConfigurationByName(launchConfigurationName: stri
             launchConfigAsInSettings.push(currentLaunchConfiguration);
             // Push into the processed 'in-memory' launch configurations array as well.
             launchConfigurations.push(currentLaunchConfiguration);
-            // Avoid updating the launchConfigurations array in settings.json for regression tests.
-            if (process.env['MAKEFILE_TOOLS_TESTING'] !== '1') {
```

Comment:
If we're removing this, unless there are other places we check for it, should we remove the MAKEFILE_TOOLS_TESTING env variable? 

Not sure I'm necessarily suggesting this, just asking. It might be just as wise to leave it in case we need it in the future.

Reply:
We still use it to disable other things in testing mode.